chainer/variable.py

```diff
@@ -511,6 +511,10 @@ class Variable(object):
     # TODO(hvy): Avoid modifying this variable from outside this class.
     _chainerx_fallback_array = None
 
+    _data = None  # type: tp.List[tp.Optional[types.NdArray]]
```

Comment:
I think it's better to place these attribute declarations before ChainerX-specific attributes.